controllers/types.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+// Config is a type to pass configuration from flags to controllers.
+type Config struct {
```

Comment:
I am not certain about this. Introducing a Config struct seemed okay in case we want to add more such flags in the future. Currently just passing the string around would also be fine.

Does anyone have any strong opinions here?

Reply:
TBD this seems overkill for a single flag...

Reply:
I changed it to string for now, still fine to do either.